posix.mak

```diff
@@ -269,12 +269,10 @@ dlangspec.tex : $(DMD) $(DDOC) latex.ddoc dlangspec-consolidated.d
 	$(DMD) -Df$@ $(DDOC) latex.ddoc dlangspec-consolidated.d
 
 # Run twice to fix multipage tables and \ref uses
-dlangspec.dvi : dlangspec.tex
-	latex $^
-	latex $^
-
-$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.dvi
-	dvipdf $^ $@
+$(DOC_OUTPUT_DIR)/dlangspec.pdf : dlangspec.tex
+	pdflatex -draftmode $^
+	pdflatex $^
+	-mv dlangspec.pdf $(DOC_OUTPUT_DIR)/dlangspec.pdf
```

Comment:
The `mv` also serves a second purpose, to protect against a fallacy that is common in makefiles. If the build process is interrupted, the partially-created PDF file will have an mtime newer than its dependencies, thus it will not be rebuilt.